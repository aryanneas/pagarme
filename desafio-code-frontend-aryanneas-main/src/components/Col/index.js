import styled from 'styled-components'

const Col = styled.div`
  display: ${(props) => props.size || 'flex'};
  width: 100%;
`

export default Col
