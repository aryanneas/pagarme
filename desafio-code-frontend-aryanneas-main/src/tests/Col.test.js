import React from 'react'
import { render } from '@testing-library/react'

import Col from '../components/Col'

const text = 'Hello World!'

describe('Col component', () => {
  it('should render', () => {
    const { asFragment } = render(<Col />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render children', () => {
    const { getByText } = render(
      <Col>
        <p>{text}</p>
      </Col>
    )

    getByText(text)
  })
})
