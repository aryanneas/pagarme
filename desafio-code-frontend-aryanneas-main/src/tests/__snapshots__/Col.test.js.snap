// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col component should render 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ dpePDy"
  />
</DocumentFragment>
`;
